qa/os/src/test/java/org/elasticsearch/packaging/util/Archives.java

```diff
@@ -267,6 +263,16 @@ private static void verifyDefaultInstallation(Installation es, Distribution dist
         Shell sh,
         String keystorePassword,
         boolean daemonize
+    ) {
+        return runElasticsearchStartCommand(installation, sh, keystorePassword, List.of(), daemonize);
+    }
+
+    public static Shell.Result runElasticsearchStartCommand(
```

Comment:
There are only 2 uses of the existing method. Please modify them to pass in List.of() rather than creating another variant.